src/test/java/core/basesyntax/service/WriterServiceImplTest.java

```diff
@@ -4,16 +4,16 @@
 
 import core.basesyntax.service.impl.WriterServiceImpl;
 import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 
 class WriterServiceImplTest {
-    private WriterService writerService;
+    private static WriterService writerService;
+    private static String report;
     private String pathToFile;
-    private String report;
 
-    @BeforeEach
-    void setUp() {
+    @BeforeAll
+    static void setUp() {
         writerService = new WriterServiceImpl();
         report = "fruit,quantity" + System.lineSeparator()
```

Comment:
I don't think that this is a mistake but you probably just could declare report as a constant

Reply:
I did this because I thought initializing it as a regular constant would not be correct since I use 2 System.lineSeparator() here.